langchain/document_loaders/grobid.py

```diff
@@ -0,0 +1,59 @@
+import requests
+from bs4 import BeautifulSoup
```

Comment:
Remove. Make this a local import inside the class definition, which you already have. See [here](https://github.com/hwchase17/langchain/pull/6693) for reference.